docs/content/preview/drivers-orms/java/ysql-jdbc-ssl.md

```diff
@@ -0,0 +1,204 @@
+---
```

Comment:
Roll this into the postgres-jdbc page; the only difference is 2 extra bullets of setup under CRUD operations, this page just isn't needed. The jdbc even has the SSL parameters already